app/core/builder/blueprint.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+# For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+from typing import FrozenSet, Union, List, Iterable
+
+from astropy.time import TimeDelta
+import astropy.units as u
+
+from lucupy.minimodel.observation import ObservationClass
+from lucupy.minimodel.program import ProgramTypes
+from lucupy.minimodel.semester import Semester, SemesterHalf
+from lucupy.minimodel.site import Site, ALL_SITES
+
+from app.config import config, ConfigurationError
+from app.core.components.optimizer.dummy import DummyOptimizer
+
+class Blueprint:
```

Comment:
This class was created with the purpose to enclose the parser logic special for each and not abuse the global nature of `config`. Also in the future for settings modification we can use `Blueprint` classes to acts as mutable objects and reconfigure the specified components without needing to deploy a new scheduler (or restarting the system entirely).